themes/root/templates/HelpTranslations/en/markdown.phtml

```diff
@@ -0,0 +1,67 @@
+<h1>Markdown and emojis</h1>
+
+<ul class="HelpMenu">
+  <li><a href="#markdown">Markdown</a></li>
+  <li><a href="#emojis">Emojis</a></li>
+</ul>
+
+<dl class="Content">
+  <dt><a id="markdown"></a>Markdown</dt>
+  <dd>
+    <p>
+      Markdown is a simple way to format text using symbols. You can add formatting to your text without needing
+      to learn complicated coding.
+      <b>Please see <a href="https://www.markdownguide.org/basic-syntax/" target="_blank">this guide</a>
+        for a detailed explanation of all markdown features.</b> Some examples are given below.
+    </p>
+    <p>
+      If you want to make a word or phrase stand out, you can put asterisks (*) or underscores (_) around it.
+      One asterisk or underscore on each side will make the text <i>italic</i>, while two will make it <b>bold</b>.
+    </p>
+    <pre class="code">
+      _italic text_
+      **bold text**</pre>
+    <p>
+      For headers put a hashtag (#) in front of a line to make it a header. The more hashtags you use, the
+      smaller the header becomes. So, one hashtag is a big header, while three hashtags are a small one.
+    </p>
+    <pre class="code">
+      #Heading1
+      ##Heading2
+      ###Heading3</pre>
+    <p>
+      For a bullet list, start each line with a dash (-) or an asterisk (*).
+      If you want a numbered list, start each line with a number followed by a period.
+    </p>
+    <pre class="code">
+      1. First item
+      2. Second item
+      3. Third item
+      4. Fourth item</pre>
+    <p>
+      If you want to link to a website, put the text you want to display in square brackets ([]), followed by the
+      website address in parentheses ().
+    </p>
+    <pre class="code">
+      [VuFind](https://vufind.org)</pre>
```

Comment:
Please add the ® symbol to the end of VuFind here -- we try to consistently include the trademark symbol when displaying the name in the user interface.